### PR TITLE
add css style for code inside markdown

### PR DESCRIPTION
**Pull request by Moelf**

### CODE REVIEW

**Comment by Moelf** [comment]
after:
image

**Comment by fonsp** [comment]
Can you make it prettier? Experiment with padding, border, rounded corners, an accent color

**Comment by Moelf** [comment]
before:
image

after:
image

File: frontend/editor.css [hunk 1]
```diff
@@ -167,6 +167,12 @@ pluto-output code {
     font-size: 0.75rem;
 }
 
+pluto-output .markdown code {
```
Comment (fonsp) [inline frontend/editor.css:170]: What happens if you remove the `.markdown` selector?

Comment (Moelf) [reply frontend/editor.css:170]: it will influence the normal output, say cell:
```
x = 3
```
the `3` will be displayed as if it's in markdown.

**Comment by fonsp** [comment]
I will reformat this so that we don't need the `.markdown` selector:
```julia
html"""
<p>Hello <code>world</code></p>
"""
```
should look identical to 
```julia
md"""
Hello `world`
"""
```

**Comment by fonsp** [comment]
I think it's `:not(.inline)` but i'm not sure

**Comment by Moelf** [comment]
close, turns out it's `.inline_output` ;)
image

**Comment by pankgeorg** [comment]
Why is the `border-radius` so big?
Github seems to use `6px`  (both `<code>` and `<pre>`)
Slack seems to use `3px` for `<code>` and `4px` for `<pre>`
[Documenter.jl](https://juliadocs.github.io/Documenter.jl/stable/) seems to use `0` with a `0.1em`padding and a bigger (`.875em vs .75em`) font size.

**Comment by fonsp** [comment]
Because it's pretty 🌈

There's no way to discuss these things objectively, but note that we are not trying to match the style of either documenter, slack or github. (Especially not slack, yuck!)

**Comment by pankgeorg** [comment]
That is not what I'm suggesting (to match something) -- but I'm pretty sure they've had teams of UX designers that decided against big border-radiuses (same is true for Jupyter btw - `2px`).
Even though it _is_ pretty, being pretty is not it's role - I think big/pretty --> distracting, that's why I would suggest a `6px` or less for both cases. Up to you, of course.

**Comment by Moelf** [comment]
while I more or less agree with you, can we just get this merged first so I can at least SEE the code instead of NOT SEE them?

**Comment by fonsp** [comment]
As Oscar Wilde put it:

> A fashion is merely a form of ugliness so absolutely unbearable that we have to alter it every six months!

**Pull request merged**